### PR TITLE
Remaining documentation symbols.

### PR DESCRIPTION
**Pull request by alxhub**

### CODE REVIEW

File: modules/angular2/http.ts [hunk 1]
```diff
@@ -291,6 +293,8 @@ export const JSONP_PROVIDERS: any[] = [
 ];
 
 /**
+ * See {@link JSONP_PROVIDERS} instead.
```
Comment (jeffbcross) [inline modules/angular2/http.ts]: Recapping IRL convo with @alxhub : Alex is going to ask docs infrastructure team to add a note about any API with deprecated decorator to indicate that it is deprecated in the output.

Comment (alxhub) [inline modules/angular2/http.ts]: Filed https://github.com/angular/angular.io/issues/501 for this.

File: modules/angular2/src/core/reflection/reflection.ts [hunk 2]
```diff
@@ -4,4 +4,8 @@ import {Reflector} from './reflector';
 export {Reflector, ReflectionInfo} from './reflector';
 import {ReflectionCapabilities} from './reflection_capabilities';
 
+/**
+ * The {@link Reflector} used internall in Angular to access metadata
```
Comment (robwormald) [inline modules/angular2/src/core/reflection/reflection.ts]: spelling : internally

**Comment by robwormald** [comment]
LGTM other than the one spelling errar.

**Comment by IgorMinar** [comment]
@alexeagle can you please rebase, fix the typo and merge please?

**Comment by alxhub** [comment]
@IgorMinar @alexeagle 

Typo fixed and rebased, ready for merge!

**Pull request closed without merging**